go-controller/pkg/node/node.go

```diff
@@ -546,7 +546,7 @@ func configureSvcRouteViaBridge(bridge string) error {
 		if err != nil {
 			return fmt.Errorf("unable to find gateway IP for subnet: %v, found IPs: %v", subnet, gwIPs)
 		}
-		err = util.LinkRoutesAdd(link, gwIP[0], []*net.IPNet{subnet})
+		err = util.LinkRoutesAdd(link, gwIP[0], []*net.IPNet{subnet}, config.Default.MTU)
```

Comment:
And this would affect also nodeport / externalip services right?

Reply:
right in shared gw mode, any packet coming through host kernel towards nodeport will get DNAT'ed to cluster IP. External IP may be a little different depending on if the ip exists on the node itself or not.  I cant remember the details there, but we already handle that today. Either way with external ip not on the node it will get DNAT'ed to the cluster ip and hit this route.